### PR TITLE
F fix/style

### PR DESCRIPTION
**Pull request by valeskus**

### CODE REVIEW

**Review by MAKARD — DISMISSED** [review]

File: src/UI/normalize.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Dimensions, PixelRatio, Platform } from 'react-native';
+
+const {
+    width: SCREEN_WIDTH,
+  } = Dimensions.get('window');
```
Comment (MAKARD) [inline src/UI/normalize.ts:6]: format

File: src/UI/normalize.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  const scale = SCREEN_WIDTH / 320;
```
Comment (MAKARD) [inline src/UI/normalize.ts:8]: format

File: src/UI/normalize.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+export function normalize (size: number){
+    const newSize = size * scale;
+    if (Platform.OS === 'ios') {
+
+      return Math.round(PixelRatio.roundToNearestPixel(newSize));
+    } else {
+      return Math.round(PixelRatio.roundToNearestPixel(newSize)) - 2;
```
Comment (MAKARD) [inline src/UI/normalize.ts]: name that constant according to its purpose

File: src/components/Search/Search.tsx [hunk 4]
```diff
@@ -32,6 +33,7 @@ export function Search({ onSearch }: Props): JSX.Element {
         value={searchTerm}
         ref={searchInputRef}
         autoCapitalize="none"
+        placeholderTextColor = {Colors.borderTextSecondary}
```
Comment (MAKARD) [inline src/components/Search/Search.tsx]: format

**Review by MAKARD — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 5]
```diff
@@ -52,17 +54,10 @@ export function App(): JSX.Element {
 
   return (
     <Provider store={store}>
+      <StatusBar backgroundColor={Colors.background} barStyle={'dark-content'} />
```
Comment (valeskus) [inline src/App.tsx]: barStyle="..."

File: src/UI/normalize.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+const {
+  width: SCREEN_WIDTH,
+} = Dimensions.get('window');
+
+const ANDROID_NORMALIZE = 2;
+
+const scale = SCREEN_WIDTH / 320;
+
+export function normalize(size: number) {
```
Comment (valeskus) [inline src/UI/normalize.ts]: write unit tests for this function

File: src/screens/RecipeDetails/RecipeDetails.tsx [hunk 7]
```diff
@@ -56,6 +56,9 @@ export function RecipeDetails(): JSX.Element {
 
   return (
     <View style={styles.detailsScreenContainer}>
+      <StatusBar backgroundColor="transparent" barStyle={'dark-content'}
```
Comment (valeskus) [inline src/screens/RecipeDetails/RecipeDetails.tsx]: barStyle="..."

File: src/screens/RecipeDetails/components/Header/Header.tsx [hunk 8]
```diff
@@ -24,7 +26,7 @@ export function Header({ scrollYRef }: Props): JSX.Element {
   return (
     <SafeAreaView style={styles.header}>
       <Animated.View style={[styles.headerWrapper, { opacity: headerOpacity }]} />
-      <View style={styles.headerLeftButton}>
+      <View style={[styles.headerLeftButton, { marginTop: STATUSBAR_HEIGHT }]}>
```
Comment (valeskus) [inline src/screens/RecipeDetails/components/Header/Header.tsx]: should be done by https://reactnative.dev/docs/statusbar#currentheight-android

File: src/screens/RecipeDetails/components/RecipeDetailsSkeleton/RecipeDetailsSkeleton.tsx [hunk 9]
```diff
@@ -1,7 +1,13 @@
 import React from 'react';
 import ContentLoader, { Circle, Rect } from 'react-content-loader/native';
+import { NativeModules, Platform, StatusBar } from 'react-native';
 
 import { Colors } from '@UI/Colors';
+import { normalize } from '@UI/normalize';
+
+const { StatusBarManager } = NativeModules;
+
+const STATUSBAR_HEIGHT = Platform.OS === 'ios' ? normalize(60) : StatusBarManager.HEIGHT + 25;
```
Comment (valeskus) [inline src/screens/RecipeDetails/components/RecipeDetailsSkeleton/RecipeDetailsSkeleton.tsx]: https://reactnative.dev/docs/statusbar#currentheight-android

File: src/screens/RecipeDetails/components/RecipeDetailsSkeleton/RecipeDetailsSkeleton.tsx [hunk 10]
```diff
@@ -12,8 +18,11 @@ export function RecipeDetailsSkeleton(): JSX.Element {
       width={'100%'}
       height={'100%'}
     >
+      <StatusBar backgroundColor="transparent" barStyle={'dark-content'}
```
Comment (valeskus) [inline src/screens/RecipeDetails/components/RecipeDetailsSkeleton/RecipeDetailsSkeleton.tsx]: barStyle="..."

File: src/UI/normalize.test.ts [hunk 11]
```diff
@@ -0,0 +1,35 @@
+import { Dimensions } from 'react-native';
+
+import { normalize } from './normalize';
+
+describe('normalize', () => {
+    beforeAll(() => {
+        jest.spyOn(Dimensions, 'get');
+    });
+
+    it('should return the same size as we send to normalize the function when the width is 320', () => {
```
Comment (MAKARD) [inline src/UI/normalize.test.ts]: to normalize the function -> to the normalize function

File: src/UI/normalize.test.ts [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+            width: width320,
+            height: 0,
+            scale: 0,
+            fontScale: 0,
+        });
+
+        expect(normalize(17)).toBe(17);
+    });
+
+    it('should return a size greater than we send to normalize the function when the width is more than 320', () => {
```
Comment (MAKARD) [inline src/UI/normalize.test.ts]: to normalize the function -> to the normalize function

File: src/screens/RecipeDetails/RecipeDetails.tsx [hunk 13]
```diff
@@ -56,6 +56,9 @@ export function RecipeDetails(): JSX.Element {
 
   return (
     <View style={styles.detailsScreenContainer}>
+      <StatusBar backgroundColor="transparent" barStyle="dark-content"
```
Comment (MAKARD) [inline src/screens/RecipeDetails/RecipeDetails.tsx]: format either all props in one line or each prop on a separate line

File: src/screens/RecipeDetails/components/RecipeDetailsSkeleton/RecipeDetailsSkeleton.tsx [hunk 14]
```diff
@@ -12,8 +14,11 @@ export function RecipeDetailsSkeleton(): JSX.Element {
       width={'100%'}
       height={'100%'}
     >
+      <StatusBar backgroundColor="transparent" barStyle="dark-content"
+        translucent={true}
+      />
       <Rect x="0" y="0" rx="0" ry="0" width="100%" height="100%" />
-      <Circle cx={'9.5%'} cy={80} r={25} />
+      <Circle cx={'9.5%'} cy={(StatusBar.currentHeight && StatusBar.currentHeight + 25)} r={normalize(25)} />
```
Comment (MAKARD) [inline src/screens/RecipeDetails/components/RecipeDetailsSkeleton/RecipeDetailsSkeleton.tsx]: what if StatusBar.currentHeight is null/undefined/NaN/false etc

File: src/screens/RecipesList/components/RecipesListMessage/styles.ts [hunk 15]
```diff
@@ -6,7 +6,7 @@ import { Fonts } from '@UI/Fonts';
 export const styles = StyleSheet.create({
   textMessageContainer: {
     width: '100%',
```
Comment (MAKARD) [inline src/screens/RecipesList/components/RecipesListMessage/styles.ts]: width/height and flex in the same style leads to only flex style active

**Review by MAKARD — CHANGES_REQUESTED** [review]

**Review by MAKARD — DISMISSED** [review]

**Review by MAKARD — APPROVED** [review]

**Pull request merged**